lib/_dispatch_widgets.fish

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env fish
+
+if test (uname) = "Darwin"
+	set -g icons 🚏 ⏱ 🚄 💨
```

Comment:
Might be worth thinking about if we can make these configurable

Reply:
What I'd like to do ultimately is have the widget system output either descriptors of some sort, or variables, so you can do your own theming. (I don't think it's entirely fair to force my preference for Font Awesome on everybody.)

I'm just not entirely sure what that would actually look like... #4

Reply:
Honestly? For now, could we just go with always making it emoji?